scripts/test_line_bot.py

```
@@ -0,0 +1,233 @@
+#!/usr/bin/env python3
+"""
+LINE Bot機能テストスクリプト
+"""
+
+import asyncio
+import sys
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)